ansible/roles/wifi-ap/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 ssid: "ConnectBox - Free Media"
+wireless_channel: 3
```

Comment:
Can you add a comment about channel 1 being broken on OPi, please, so it’s clear why we chose this default.